### PR TITLE
socialhub-fa:1.0.0

### PR DESCRIPTION
**Pull request by Bi0T1N**
I am submitting
- [x] a new package
- [ ] an update for a package

Description:
The `socialhub-fa` package is designed to help you create your curriculum vitae (CV). It allows you to easily reference your social media profiles with the typical icon of the service plus a link to your profile.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

### CODE REVIEW

**Comment by reknih** [comment]
Thank you for your package submission! We have to ask you to change the name of this package since `social` could be the natural name for packages with these color sets. By taking up this name, the potential for other packages to improve upon this one is reduced. We can continue merging this once you push a new name!

**Comment by Bi0T1N** [comment]
> Thank you for your package submission! We have to ask you to change the name of this package since `social` could be the natural name for packages with this functionality. By taking up this name, the potential for other packages to improve upon this one is reduced. We can continue merging this once you push a new name!

Would `social-fa` (`fa` for Font Awesome) be accepted as package name? Or is it preferred to have long versions for package names: `social-font-awesome`?

**Comment by reknih** [comment]
Both names would be acceptable. Meanwhile, have you considered something less normative along the lines of `socialite` or `influencer`?

**Comment by Bi0T1N** [comment]
> Both names would be acceptable. Meanwhile, have you considered something less normative along the lines of `socialite` or `influencer`?

I don't have a firm opinion on the name. I've simply taken what I'm used to from [LaTeX](https://github.com/xdanaux/moderncv/blob/master/examples/template.tex#L35).
Some other ideas:
- socialhub
- references (probably too generic)
- socialreferences
- internetreferences
- websiterefs

I'm open to all kinds of suggestions. :smile:

**Comment by reknih** [comment]
Socialhub would work!

**Comment by Bi0T1N** [comment]
I've renamed to package to `socialhub-fa` (`fa` is the short version of *font awesome*)

File: packages/preview/socialhub-fa/1.0.0/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+### Linux
+1. [Download Font Awesome for Desktop](https://fontawesome.com/download)
+2. Unzip the file
+3. Switch into the `otfs` folder within the unzipped folder
+4. Run `mkdir -p /usr/share/fonts/truetype/`
+5. Run `install -m644 'Font Awesome 6 Brands-Regular-400.otf' /usr/share/fonts/truetype/`
+6. Unfortunately not all brands are included in the brands font file, so you must also run `install -m644 'Font Awesome 6 Free-Regular-400.otf' /usr/share/fonts/truetype/`
+
+## Usage
+### Using the Typst's package manager
```
Comment (laurmaedje) [inline packages/preview/socialhub-fa/1.0.0/README.md:21]:
```suggestion
### Using Typst's package manager
```

File: packages/preview/socialhub-fa/1.0.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "socialhub-fa"
+version = "1.0.0"
+repository = "https://github.com/Bi0T1N/typst-socialhub-fa"
+entrypoint = "socialhub-fa.typ"
+authors = ["Nico Neumann (Bi0T1N)"]
+license = "MIT"
+description = "A Typst library for Social Media references with icons based on Font Awesome."
+keywords = ["social", "media", "icons", "Font Awesome"]
+exclude = ["examples/*.pdf"]
```
Comment (laurmaedje) [inline packages/preview/socialhub-fa/1.0.0/typst.toml:10]: any particular reason for not excluding the whole examples folder? would be simpler.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**